microsetta_public_api/api/taxonomy.py

```diff
@@ -247,3 +247,18 @@ def ranks_specific(dataset, resource, sample_id):
     payload['Taxa-order'] = order
 
     return jsonify(payload), 200
+
+
+def rare_unique(dataset, resource, sample_id, rare_threshold=0.1):
+    taxonomy_repo = _get_taxonomy_repo(dataset)
+
+    error_response = _check_resource_and_missing_ids(taxonomy_repo,
+                                                     [sample_id, ],
+                                                     resource)
+    if error_response:
+        return error_response
```

Comment:
Can you fix the pattern here, as per discussions in https://github.com/biocore/microsetta-public-api/pull/89#discussion_r568072671 , https://github.com/biocore/microsetta-public-api/pull/73/files#r540501770 , and #39 ?

Reply:
https://github.com/biocore/microsetta-public-api/pull/89/commits/15f86a293b3807505ac54747f92e511cf37d0c97#diff-775748170ea501e093e314841532c206c8104b1477f8502e31368958b60790f5R70-R83
`microsetta_public_api/api/taxonomy.py` in that diff shows the type of change.